app/bundles/ApiBundle/Config/config.php

```diff
@@ -125,7 +125,13 @@
         'forms' => [
```

Comment:
Please remove form type aliases from lines 129, 133 and 173.
They are deprecated.